tools/sigs/generator/README.md

```diff
@@ -0,0 +1,128 @@
+# SIG Doc builder
+
+This was originally copied from the [`kubernetes/community`](https://github.com/kubernetes/community/tree/master/generator) repo.
+
+This folder contains scripts to automatically generate documentation about the
+different Special Interest Groups (SIGs), Working Groups (WGs)
+and Committees of Kubernetes. The authoritative
+source for SIG information is the [`sigs.yaml`](/sigs.yaml) file in the project root.
+All updates must be done there.
+
+The schema for this file should be self explanatory. However, if you need to see all the options, check out the generator code in `app.go`.
+
+The documentation follows a template and uses the values from [`sigs.yaml`](/sigs.yaml):
+
+- Header: [`header.tmpl`](header.tmpl)
+- List: [`list.tmpl`](list.tmpl)
+- SIG README: [`sig_readme.tmpl`](sig_readme.tmpl)
+- WG README: [`wg_readme.tmpl`](wg_readme.tmpl)
+- Committee README: [`committee_readme.tmpl`](committee_readme.tmpl)
+
+**Time Zone gotcha**:
```

Comment:
Using UTC removes the issue of daylight savings, and the KubeVirt calendar updates times depending on the user/browser location.